thirdeye/thirdeye-frontend/app/helpers/manage-alert-utils.js

```diff
@@ -4,18 +4,17 @@ import moment from 'moment';
 /**
  * Handles types and defaults returned from eval/projected endpoints
  * @param {Number|String} metric - number or string like 'NaN', 'Infinity'
- * @param {Boolean} allowDecimal - should this metric be shown as whole number?
+ * @param {Boolean} isPercentage - shall we treat this as a % or a whole number?
  * @returns {Object}
  */
-export function formatEvalMetric(metric, allowDecimal = false) {
-  let shown = 'N/A';
-  if (!isNaN(metric)) {
-    if (allowDecimal) {
-      shown = (Number(metric) === 1 || Number(metric) === 0)
-        ? metric * 100
-        : (metric * 100).toFixed(1);
+export function formatEvalMetric(metric, isPercentage = false) {
+  const isWhole = Number(metric) % 1 === 0;
```

Comment:
I believe `Number.isInteger` does the same thing and is built-in
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger

Reply:
fixed in https://github.com/linkedin/pinot/pull/2351